src/components/MessageListItem.react.js

```diff
@@ -56,6 +57,8 @@ class MessageListItem extends React.Component {
     let {message} = this.props;
     let stringWithLinks = this.props.message.text;
     let replacedText = parseAndReplace(stringWithLinks);
+    let messageContainerClasses  = cx('message-container '
```

Comment:
But how can we add dynamic style of that class, as we don't know the class name beforehand, it'll be dynamic right ? depending upon the user, or am I going in some other direction?.